### DISCUSSION TITLE
Happy 10th Anniversary (of open sourcing)

### DISCUSSION THREAD

**Discussion by akleeman** [pydata/xarray#8462] [General]
10 years ago today @ebrevdo and I obtained permission to open source what had been an internal package called `scidata` (that @danfrankj and I started a few years earlier). Main goal at that point: to provide a consistent interface for accessing weather data stored in a variety of formats.

Pretty crazy how far it's come since!

https://github.com/pydata/xarray/commit/08f8f29736a3e18d90f9f287b85ea3d8a63c5064

**Comment by rabernat** [comment]
This is worthy of a blog post I would say.

**Reply by dcherian** [reply]
:+1: A fair many people are surprised when I say that Xarray came out of Climate and has significant private sector participation. You should tell that story :)

**Reply by rabernat** [reply]
Maybe a retrospective blog post on the Xarray blog, about some of the big milestones in the past 10 years?

**Reply by TomNicholas** [reply]
In the meantime we should at least tweet about it today!

**Reply by dcherian** [reply]
<img width="551" alt="image" src="https://github.com/pydata/xarray/assets/2448579/622c5fcf-52a0-45ef-a372-9110bc69d11b">
edits?

**Reply by TomNicholas** [reply]
Great - just needs a snazzy "happy birthday to me" gif

**Reply by dcherian** [reply]
https://twitter.com/xarray_dev/status/1725594118707986697 I think i did well on the gif :P

**Reply by TomNicholas** [reply]
Excellent choice 😁

**Comment by dcherian** [comment]
>  @ebrevdo and I obtained permission

Well thank you! 🫡

**Comment by ebrevdo** [comment]
It's been amazing watching the project and community grow around it.  xarray has really turned into something bigger than we originally imagined thanks to all of its users, contributors, and maintainers!

**Comment by TomNicholas** [comment]
Also how many names has xarray had?? `scidata` -> `polyglot` (https://github.com/pydata/xarray/commit/08f8f29736a3e18d90f9f287b85ea3d8a63c5064) -> `xray` (https://github.com/pydata/xarray/commit/f08e2ebc58e34485eccdc8e25f9deb9afc9ed4a9) -> `xarray` (https://github.com/pydata/xarray/commit/c636f511961f220c60fe141f833114590525e8bd) ?

**Comment by TomNicholas** [comment]
This is awesome thank you so much!

**Comment by Happy-fish-yu** [comment]
Hooray! I can't imagine the inconvenience of handling NetCDF format data without this marvelous package!

**Comment by Shaw-Hong** [comment]
Thank You Very Much !!!
